integration-test/grpc_create_model.js

```diff
@@ -56,10 +56,10 @@ export function CreateModel() {
         let currentTime = new Date().getTime();
         let timeoutTime = new Date().getTime() + 120000;
         while (timeoutTime > currentTime) {
-            let res = client.invoke('vdp.model.v1alpha.ModelPublicService/GetModelOperation', {
```

Comment:
the purpose here is wait for the operation done then we check the status. If it's error, the test case failed because we expect it online. We user check status it will be offline that case then need wait to time out of error case?